lib/zipkin-tracer/excon/zipkin-tracer.rb

```diff
@@ -47,6 +53,7 @@ def response_call(datum)
     private
 
     SERVER_ADDRESS_SPECIAL_VALUE = '1'.freeze
+    STATUS_ERROR_REGEXP = /\A(4.*|5.*)\z/.freeze
```

Comment:
you could also put 1.x here, eventhough it is unlikely.. or reverse it to things that are not 2xx and 3xxx